.github/workflows/ci.yaml

```diff
@@ -1,7 +1,11 @@
+name: 'CI'
+
 on:
+  workflow_dispatch:
```

Comment:
why is this needed?

Reply:
This is a quality of life thing. It allows you (and other people who have write access to this repo) to manually start the workflow. Can be useful sometimes.

Reply:
See here: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#running-a-workflow-on-github

Reply:
It can be, but I find that if I have a pretty loose CI policy like trigger on every push, I don't really ever have a need for this. I'd prefer to leave it as it was in #24.

Reply:
removed. :)